src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -578,9 +578,8 @@ void SystemRequest::Run() {
     msg_params[strings::file_name] = file_dst_path;
   }
 
-  if (mobile_apis::RequestType::PROPRIETARY != request_type) {
-    msg_params[strings::app_id] = (application->policy_app_id());
-  }
+  msg_params[strings::app_id] = application->hmi_app_id();
+
```

Comment:
@dtrunov have you read the HMI_API spec?. The policy appplication ID is a string whereas according to the api integer value is required